keps/sig-node/4622-topologymanager-max-allowable-numa-nodes/kep.yaml

```diff
@@ -0,0 +1,39 @@
+title: New TopologyManager Policy which configure the value of maxAllowableNUMANodes
+kep-number: 4622
+authors:
+  - "@cyclinder"
+owning-sig: sig-node
+participating-sigs: []
+status: implementable
+creation-date: "2024-05-08"
+reviewers:
+  - "@klueska"
+  - "@ffromani"
+approvers:
+  - "@sig-node-tech-leads"
+see-also: []
+replaces: []
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: beta
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.31"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  beta: "v1.31"
+  stable: "v1.32"
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: "TopologyManagerPolicyBetaOptions"
+    components:
+      - kubelet
+disable-supported: true
+
+# The following PRR answers are required at beta release
+metrics: []
```

Comment:
we do: https://github.com/kubernetes/enhancements/pull/4624#discussion_r1627545603

Reply:
Does that need to be listed here? I don't actually know what is meant to be put in this array.

Reply:
I don't recall either if we are supposed to list here the metrics involved in the change or the metrics which we plan to add. Since it will be already evident in the KEP text (multiple times) I'd leave this empty and fill if requested by PRR reviewers

Reply:
I think we should add the metric, I've seen other KEPs do the same thing.

Reply:
I will add the metric now, I'm open to remove this if the list should be empty.